src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -18,6 +10,7 @@ import InstructionsCard from './components/InstructionsCard';
 import ApplicationDataTable from './components/ApplicationDataTable';
 import ApplicantList from './components/ApplicantList';
 import HallSelection from './components/HallSelection';
+import OffCampusSection from './components/OffCampusSection';
```

Comment:
If we do not want the files I pointed out above, then this would have to go, right?

Reply:
Also from Nick's branch, #1031